src/js/components/receiving/PartialReceivingPage.jsx

```diff
@@ -261,15 +261,38 @@ const TABLE_FIELDS = {
       unitOfMeasure: {
         type: (params) => <LabelField {...params} />,
         label: 'react.partialReceiving.shippedInPo.label',
-        defaultMessage: 'Shipped (in PO)',
+        defaultMessage: 'Shipped (in PO UoM)',
         multilineHeader: true,
         flexWidth: '1',
         attributes: {
+          cellClassName: 'text-right',
           showValueTooltip: true,
         },
-        getDynamicAttr: (props) => ({
-          hide: !props.values?.isShipmentFromPurchaseOrder,
-        }),
+        getDynamicAttr: ({ values, parentIndex, rowIndex }) => {
+          const shipmentItem = _.get(
+            values,
+            `containers[${parentIndex}].shipmentItems[${rowIndex}]`,
+            {},
+          );
+          const packsRequested = _.round(shipmentItem?.packsRequested, 1);
```

Comment:
Is this rounding correct? Have you confirmed that behavior? This would give the same result for PK/6 and shipped 1 in each, and for PK/5 and shipped 1 in each, which is 0.2. Feels like rounding to two decimal places would be more accurate.

Reply:
Yeah on second thought I thing this should be 2 decimal places, feels like quarters (`.25`) might be a potential common case. In the current version this would be rounded up to `0.3`.